src/daemon-base/Dockerfile

```diff
@@ -21,8 +21,8 @@ RUN \
     #   download and install packages from web, cleaning any files as you go.
     # Installs should support install of ganesha for luminous without installing for jewel/kraken.
     __DOCKERFILE_INSTALL__ && \
-    # Clean container, starting with record of current size
-    INITIAL_SIZE=$(du -sm / 2>/dev/null | awk '{print $1}') && \
+    # Clean container, starting with record of current size (strip / from end)
+    INITIAL_SIZE="$(bash -c 'sz="$(du -sm / 2>/dev/null || true)" ; echo "${sz%*/}"')" && \
```

Comment:
Instead of hiding stderr, I think we simply need to exclude `/proc` from `du`'s calculation.
I just tried in a container and I nicely get the size:

```
[root@ceph-nano-e-faa32aebf00b /]# du -sm /
du: cannot access '/proc/1558/task/1558/fd/4': No such file or directory
du: cannot access '/proc/1558/task/1558/fdinfo/4': No such file or directory
du: cannot access '/proc/1558/fd/3': No such file or directory
du: cannot access '/proc/1558/fdinfo/3': No such file or directory
1179    /
[root@ceph-nano-e-faa32aebf00b /]# du -sm --exclude=/proc /
1179    /
```